go/bundle/bundle_test.go

```diff
@@ -0,0 +1,58 @@
+package bundle_test
+
+import (
+	"bytes"
+	"net/http"
+	"net/url"
+	"reflect"
+	"testing"
+
+	. "github.com/WICG/webpackage/go/bundle"
```

Comment:
can we use bundle.Bundle instead of `.` import?

Reply:
I think using a dot in a test package is common, but I'm fine with both.

Reply:
OK. Wasn't a strong preference